packages/ui-components/src/components/QrCode/QrCodeVisual.tsx

```diff
@@ -35,6 +38,11 @@ const Error = styled.div`
   text-align: center;
 `;
 
+const StyledParagraph = styled.p`
+  color: white;
```

Comment:
I think we could use the `Typography` component from mui for this, and we would ideally use theme variables for colours as well (if not already done by Typography component).